bin/test-with-real-projects.sh

```diff
@@ -13,3 +13,11 @@ git checkout 302faed # release 8.3.2
 composer install
 ~/project/build/psalm.phar --config=.psalm/config.xml --monochrome --show-info=false
 ~/project/build/psalm.phar --config=.psalm/static-analysis.xml --monochrome
+
+cd /tmp/testing-with-real-projects
+
+git clone git@github.com:roave/you-are-using-it-wrong.git
+cd you-are-using-it-wrong
+composer install
```

Comment:
This seems to run on a PHP 7.2+ circle-ci instance, so it fails:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - This package requires php ^7.3.0 but your PHP version (7.2.22) does not satisfy that requirement.
```

Reply:
Feel free to bump the version it runs on!